src/redis-benchmark.c

```diff
@@ -125,8 +126,20 @@ static struct config {
     int enable_tracking;
     pthread_mutex_t liveclients_mutex;
     pthread_mutex_t is_updating_slots_mutex;
+    int requests_emitted;
```

Comment:
I think this needs to be re-zeroed per call to `benchmark()`, In any case running the code like this:
`src/redis-benchmark --qps 2000 -n 10000`
Never exists because the second "mbulk_ping" benchmark never finishes.